code/Native/Unix/SerialUnix.cs

```diff
@@ -10,14 +10,18 @@ namespace RJCP.IO.Ports.Native.Unix
 
     internal class SerialUnix : INativeSerialDll
     {
+#if !NETSTANDARD15        
```

Comment:
Don't worry - after a recompile, this type isn't in .NET Standard. https://github.com/dotnet/coreclr/issues/1605